Exercises_Java/src/SchoolGrades.java

```diff
@@ -0,0 +1,43 @@
+// . EXERCISE TO PRACTICE IF-ELSE
+
+public class SchoolGrades {
+    private int age;
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+
+    public SchoolGrades (int age) {
+        setAge(age);
+    }
+
+
+    /*Control_sentences using conditions
+     * First Example:  IF-ELSE IF- ELSE
+     *  US-GRADES */
+
+    public void knowingMyGrade() {
+        if (getAge() < 3) {
```

Comment:
Veo que estas usando getters and setters lo cual esta bien y probablemente veamos ese tema mas adelante para la ocultacion y manejo externo de informacion, aqui la unica sugerencia que te hago es que tanto el metodo get como el set se usa para interactuar fuera de la clase, para esta comparacion puedes usar el valor que ya guardaste en tu variable age.